app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,21 @@
+class InvitationsController < ApplicationController
+  def create
+    invitation = Invitation.new(model_param)
+
+    if invitation.create_and_send
+      redirect_to(
+        root_path,
+        notice: "邀請信已經送到您的信箱囉！很期待見到你！")
+    else
+      redirect_to(
+        root_path,
+        alert: "唉呦！出錯了... #{invitation.errors.full_messages}.")
+    end
+  end
+
+  private
+
+  def model_param
```

Comment:
I found both indentation style used in [Rails](https://github.com/rails/rails)

Personally, I prefer 
```` ruby
class C
  private

  def f
  end
end
````